### DISCUSSION TITLE
About the usage of zilliztech/milvus-backup

### DISCUSSION THREAD

**Discussion by bingbb47** [milvus-io/milvus#21364] [Q&A and General discussion]
could i create a backup from 2.1.4 cluster, and restore on 2.2.x cluster?

**Comment by xiaofan-luan** [comment]
> could i create a backup from 2.1.4 cluster, and restore on 2.2.x cluster?

It should be working.

**Reply by bingbb47** [reply]
-  i want to duplicate a collection from cluster A to 2.2.x cluster B, as steps:
  1. use zilliztech/milvus-backup create a buckup
  2. copy the buckup folder from A's minio to B's minio
  3. restore buckup in cluster B
- Then I realized the backup folder only contains the meta..

**Reply by xiaofan-luan** [reply]
could you clarify your problem?
for backup data should also be restored
